packages/flutter/lib/src/rendering/semantics_event.dart

```diff
@@ -0,0 +1,33 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// An event that can be send by the application to notify interested listeners
+/// that something happened to the user interface (e.g. a view scrolled).
+///
+/// These events are usually interpreted by assistive technologies to give the
+/// user additional clues about the current state of the UI.
+abstract class SemanticsEvent {
+
+  /// Initializes internal fields.
+  ///
+  /// `type` is a string that identifies this class of [SemanticsEvent]s.
```

Comment:
We normally use `[type]` if it matches a field, so that it'll link to it (eventually, there's a dartdoc bug right now)